library/translation.inc.php

```diff
@@ -301,4 +301,27 @@ function mb_strpad($input, $length, $pad = ' ', $type = STR_PAD_RIGHT, $charset
 
 return $output;
 }
-?>
+
+
+/**
+ * Returns language direction
+ * @param int $lang_id language code
+ * @return string 'ltr' 'rtl'
+ * @author Amiel <amielel@matrix.co.il>
+ */
+function getLanguageDir($lang_id) {
+    //static $rtl_lang_codes = array('he','ar');
+
+    // validate language id
+    $lang_id = empty($lang_id) ? $lang_id : (int)$lang_id;
```

Comment:
Don't really need to validate it to an int since it gets escaped in the below sql query. Could do something like(if it's empty then use the English language):
``````
$lang_id = empty($lang_id) ? 1 : $lang_id;